df-curvy-walls/module.json

```diff
@@ -1,30 +1,44 @@
 {
-	"name": "{{name}}",
-	"title": "{{title}}",
-	"description": "{{description}}",
-	"version": "{{version}}",
+	"name": "df-curvy-walls",
+	"title": "DF Curvy Walls",
+	"description": "Creates wall segments along a Bezier Cubic or Quadratic curve or circle",
+	"version": "1.3.4",
 	"author": "flamewave000#6766",
 	"minimumCoreVersion": "0.7.5",
 	"compatibleCoreVersion": "0.7.9",
-	"esmodules": "{{sources}}",
-	"styles": "{{css}}",
+	"esmodules": [
+		"src/BezierControl.js",
+		"src/BezierToolBar.js",
+		"src/df-wall-curves.js",
+		"src/lib.js",
+		"src/tools.js",
+		"src/lib/bezier.js",
+		"src/tools/BezierTool.js",
+		"src/tools/CircleTool.js",
+		"src/tools/CubicTool.js",
+		"src/tools/QuadTool.js",
+		"src/tools/ToolInputHandler.js"
+	],
+	"styles": [
+		"css/segment-controls.css"
+	],
 	"languages": [
 		{ "lang": "en", "path": "lang/en.json", "name": "English" },
 		{ "lang": "fr", "path": "lang/fr.json", "name": "French" },
 		{ "lang": "es", "path": "lang/es.json", "name": "Español" },
-		{ "lang": "ja", "path": "lang/ja.json", "name": "日本語" }
+		{ "lang": "ja", "path": "lang/ja.json", "name": "日本語" },
+		{ "lang": "pt-BR", "path": "lang/pt-BR.json", "name": "Português (Brasil)"}
```

Comment:
Please remove all other changes in the `df-curvy-walls/module.json` file except for these two lines in the languages part.